docs/dockstore-logo-usage-guidelines.rst

```diff
@@ -0,0 +1,129 @@
+  :width: 256
+
+.. |Icon Blue| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-blue.png
+  :width: 64
+
+.. |Icon Black| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-black.png
+  :width: 64
+
+.. |Icon White| image:: /assets/images/Dockstore_Logos_and_Icons/Dockstore-icon-white.png
+  :width: 64
+
+===============================
+Dockstore Logo Usage Guidelines
+===============================
+
+About
+=====
+It is important to establish a sense of branding throughout our publicly facing, digital media. Currently, we have several themes / layouts, navigation systems, font sets, color palettes. We would like to unify our designs under a single set of standards.
+
+Color
+=====
+The official Dockstore color is:
+
+  .. list-table:: Dockstore Blue
+
+    * - |Dockstore Dot|
+      - Hex: #7986CB
+      - R: 121, G: 134, B: 203
+
+When using the Dockstore logo or icon, the acceptable colors are Dockstore Blue, black, or white as shown in the “Logo” section of this document.
+
+If the surrounding color is visually unappealing with Dockstore Blue, please use the black or white variants as appropriate.
+
+Logos
+=====
+
+  .. tabularcolumns:: |c|c|c|
+
+  .. list-table::
+    :header-rows: 1
+
+    * - Horizontal
+      - Vertical
+      - Icon
+    * - |Horizontal blue|
+      - |Vertical blue|
+      - |Icon blue|
+    * - Dockstore-logo-horizontal-blue
+      - Dockstore-logo-vertical-blue
+      - Dockstore-icon-blue
+    * - |Horizontal black|
+      - |Vertical black|
+      - |Icon black|
+    * - Dockstore-logo-horizontal-black
+      - Dockstore-logo-vertical-black
+      - Dockstore-icon-black
+    * - |Horizontal white|
+      - |Vertical white|
+      - |Icon white|
+    * - Dockstore-logo-horizontal-white
+      - Dockstore-logo-vertical-white
+      - Dockstore-icon-white
+    * - |Horizontal black blue|
+      - |Vertical black blue|
+      -
+    * - Dockstore-logo-horizontal-black-blue
+      - Dockstore-logo-vertical-black-blue
+      -
+    * - |Horizontal white blue|
+      - |Vertical white blue|
+      -
+    * - Dockstore-logo-horizontal-white-blue
+      - Dockstore-logo-vertical-white-blue
+      -
+
+A zip file containing these logos in both PNG and SVG formats can be found :download:`here </assets/images/Dockstore_Logos_and_Icons.zip>`.
+
+Use
+===
+Do:
+
+* Use the logo or icon color that works best in the context of the document, web page, or application.
+* The Dockstore logo is preferred.  Use the Dockstore icon only when space is limited or when it is clear that the reference is to Dockstore.
+* Spacing:
+  * In most cases, the logo should be of equal prominence to any other brands with which it is shown.
+  * Allow for adequate space around the logo when used in conjunction with other identifiers.
+  * Do not place other identifiers closer than the equivalent of the width of 1/2 the logo.
+
+Do Not:
+
+* Don’t place the logo on clashing colored backgrounds.
```

Comment:
if sticking with the periods, remove the "don't" at the start of the lines
Here is an example of bullet lists that need a period to complete the introductory stem sentence:

Reply:
I have the opposite interpretation of the same link. Yeah, it talks about using bullets to close off introductory stem sentences, but that's not the only situation that it cites.

> * Use a period or other full stop after every bullet that is a sentence. (Just like this bullet list you’re reading now.)
> * Use a period after bullet list that completes the opening stem sentence that introduces it.
> * Don’t use a period after bullet lists that are not complete sentences or do not complete the opening stem sentence. 

These are full sentences. The subject is implicit, but it's still a subject.

Reply:
Unless what you're saying is that the introduction of periods make them look as though they are ending off introductory stem sentences? Even that I'm not so sure about, since Do Not being capitalized doesn't follow proper sentence capitalization anyway.

Reply:
I have altered the wording in the Do & Do Not lists to make it more consistent.  I'll push it shortly.

Reply:
BTW:  I have updated the bullet list to the following.  I'll push the change once we have the broken link resolved elsewhere.

Do:

- Use the logo or icon color that works best in the context of the document, web page, or application.
- Use the Dockstore logo when space permits. Use the Dockstore icon only when space is limited or when it is clear that the reference is to Dockstore.
- Provide the Dockstore logo equal prominence to any other brands with which it is shown.
- Allow for adequate space around the logo when used in conjunction with other identifiers.

Do not:

- Place the logo on clashing colored backgrounds.
- Place the reverse logo on similar tints.
- Place any logo variation on busy backgrounds.
- Place special effects such as drop shadows on the logo.
- Skew or distort the logo.
- Alternate the color of the logo.
- Place other identifiers closer than the equivalent of the width of 1/2 the logo.

Reply:
Some comments on these Do Nots...

> the reverse logo on similar tints

I'd recommend saying white-on-transparent instead of reverse logo just for clarity's sake. And I'd argue this applies to more than just the reverse logo. It might be best to reword this as "Place the logo on backgrounds where it is hard to read"

> Alternate the color of the logo.

I'd recommend "Alter" instead of "Alternate"

> Place other identifiers closer than the equivalent of the width of 1/2 the logo.

I like this in principle, but we're being a little hypocritical here.
<img width="877" alt="Screen Shot 2022-07-07 at 4 26 12 PM" src="https://user-images.githubusercontent.com/27784612/177887730-97af3c3b-3373-4ca3-9585-5fb4bef35e5d.png">

Reply:
Yes, we are in violation of the UCSC logo usage guidelines definitely in 1 area and possibly in a second:  1) [wrong logo](https://communications.ucsc.edu/visual-design/logos-imagery/), 2) spacing requirements. 

There is an interesting conflict here and that is between the spacing requirements and the equal prominence requirement.  I would say this usage meets the equal prominence requirement.

Reply:
Well, they are guidelines :wink:
![5af1800dbd18b30ced7a98fd63f0fe1f411091a0da880f6bf4dcdd967cce8270](https://user-images.githubusercontent.com/1730679/178033929-a6e6aa75-6924-414d-9b39-3559998d02d9.jpg)

Reply:
I've removed the "Place other identifiers closer than the equivalent of the width of 1/2 the logo" guideline.